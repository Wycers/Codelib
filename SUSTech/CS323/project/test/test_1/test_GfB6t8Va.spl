int test_11610613_1() 
{
    int i = 114514;
    int j = 0;
    int c = 1 || 0;
    if(i > j){    
        return i/j;
    }          
    return i&j;        
}
