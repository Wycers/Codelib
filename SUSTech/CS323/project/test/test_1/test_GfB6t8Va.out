Error type A at Line 9: unknown lexme '&'
